addons/hot/CfgMagazines.hpp

```diff
@@ -0,0 +1,205 @@
+class CfgMagazines {
+    class 12Rnd_PG_missiles;
+    
+    // HOT1 - tandem shaped charge HEAT (anti-tank)
+    class 6Rnd_ACE_HOT1: 12Rnd_PG_missiles { // Old style vehicle magazine
```

Comment:
All ACE class shall be tagged with ACE_. A tag is always at the beginning of the variable or classname.